### DISCUSSION TITLE
Android and Chrome OS - Interactivity

### DISCUSSION THREAD

**Discussion by roborobs1023** [Ylianst/MeshCentral#6273] [Q&A]
Any plans to add the ability to control a ChromeOS or Android device via MeshCentral? That is to actually control the movements on the screen. Currently all you have is view access.

**Comment by si458** [comment]
this has already been asked in a few different issues, including the androidagent repo itself 
https://github.com/Ylianst/MeshCentralAndroidAgent/issues/17
https://github.com/Ylianst/MeshCentralAndroidAgent/issues/7

I am trying to implement it but its very difficult as we have to stick to google's rules, to get the app on the play store, otherwise its not allowed

in theory we can use ADB if your device is rooted but thats not ideal for non-rooted devices

**Reply by si458** [reply]
https://github.com/Ylianst/MeshCentral/issues/3779

**Comment by Pshemas** [comment]
implementing such option would be awesome. Currently there seems to be no viable open source solution to manage mobile devices.
In the past it seems there were - for example Headwind Remote, which used Janus Media Server:

https://headwind-remote.com/technology/

- but as they've struggled with malware they discontinued it.
